baseapp/abci.go

```diff
@@ -109,6 +109,8 @@ func (app *BaseApp) Info(req abci.RequestInfo) abci.ResponseInfo {
 
 	return abci.ResponseInfo{
 		Data:             app.name,
+		Version:          app.appVersion,
+		AppVersion:       app.protocolVersion,
```

Comment:
The naming are a bit confusing here. Should we rename the fields names in our `BaseApp` so that it's more consistent with tendermint's `ResponseInfo`?

Reply:
+1 for following Tendermint exactly, it's the path with least confusion.

It's a breaking change, because BaseApp already has a `AppVersion()` method, so a changelog entry is needed.

Reply:
Should the interface and new get/set methods in upgrade's keeper be changed to align with this? (i.e. change `getProtocolVersion` to `getAppVersion`)

Reply:
Let's maybe keep `ProtocolVersion` everywhere else, it's way clearer than AppVersion. But still rename the current baseapp.appVersion to baseapp.version